distributions/standalone/assembly-whitelist.txt

```diff
@@ -365,22 +365,23 @@
 /mule-standalone-${productVersion}/lib/opt/signpost-core-1.2.1.2.jar
 /mule-standalone-${productVersion}/lib/opt/smack-3.1.0.jar
 /mule-standalone-${productVersion}/lib/opt/smackx-3.1.0.jar
-/mule-standalone-${productVersion}/lib/opt/spring-aop-3.2.10.RELEASE.jar
-/mule-standalone-${productVersion}/lib/opt/spring-beans-3.2.10.RELEASE.jar
-/mule-standalone-${productVersion}/lib/opt/spring-context-3.2.10.RELEASE.jar
-/mule-standalone-${productVersion}/lib/opt/spring-context-support-3.2.10.RELEASE.jar
-/mule-standalone-${productVersion}/lib/opt/spring-core-3.2.10.RELEASE.jar
+/mule-standalone-${productVersion}/lib/opt/spring-aop-4.1.3.RELEASE.jar
```

Comment:
Just noted that there is a Spring 4.1.5-RELEASE. Would be good to use the latest available version of it